src/cli/README.md

```diff
@@ -0,0 +1,36 @@
+# Command Line Interface
+
+This is a simple command line interface for interacting with a Local Fulfillment app.
+
+# Usage
+
+## To start the command line interface:
+
+Run `npm run cli -- --app_path APP_PATH`, where `APP_PATH` is an absolute path to the entry point of your app (typically `bundle.js` if basing on the [Local Home SDK Sample](https://github.com/actions-on-google/smart-home-local))
+
+## To send an IDENTIFY intent
+
+Run `identify IDENTIFY --request_id REQUEST_ID --discovery_buffer DISCOVERY_BUFFER --device_id DEVICE_ID`
```

Comment:
did you consider having the tool take the identify/execute payload from stdin?

Reply:
I considered this.  My thought process was if the developer is doing testing involving copying/pasting payload JSONS to test a handler, they may as well use the automated testing interface.  (i.e. I'm not sure copy/pasting payload JSONs is a "usable" CLI).  The idea for this PR was to provide a more intuitive / hotswappable command interface for sending intents on-demand and rapidly being able to tweak the arguments without needing to worry about schema compliance.  (I'm thinking of something like Postman's UI where you can simply tweak field values without needing to paste an entire JSON per request)

Reply:
I wonder if it would make sense to have a separate tool for generating those payload and piping them to CLI.

It seems to me that a general way to craft sync response/execute request from command line arguments cloud prove itself useful outside of the context of the Local Home SDK.

What do you think?

Reply:
I think one thing to consider is the actual use case / core issues that would solved by building such a tool.  If we find that developers are repeatedly creating JSON requests (if this is something that is regularly part of numerous different development workflows), then it may help to standardize this request generation.  I could see it being integrated as a simple web tool / integrated into the developer docs.

That being said, I think it's easy enough to integrate the request generation into tools that need it, on a case-by-case basis (what I'm doing right now).  So breaking it out as a separate tool may only complicate things.

As a side note, Postman integration could be interesting.  I could imagine[ importing a postman collection that we provide](https://learning.postman.com/docs/getting-started/importing-and-exporting-data/) as a way of providing a ready-to-use template of sample requests while piggybacking off a tool's UI/UX that's pretty universally loved.